components/api/core/api_impl.bal

```diff
@@ -18,24 +18,46 @@
 import cellery_hub_api/db;
 import ballerina/http;
 
-public function createOrgs (http:Request _createOrgsReq, gen:OrgCreateRequest _createOrgsBody) returns http:Response {
-    // stub code - fill as necessary
-    http:Response _createOrgsRes = new;
-    string _createOrgsPayload = "Sample createOrgs Response";
-    _createOrgsRes.setTextPayload(_createOrgsPayload);
-	return _createOrgsRes;
+public function createOrg (http:Request createOrgReq, gen:OrgCreateRequest createOrgsBody) returns http:Response{
+    http:Response _addOrgRes = new;
+    if (createOrgReq.hasHeader(USERNAME)){
+        string userName = createOrgReq.getHeader(USERNAME);
```

Comment:
The header would contain the user UUID. Therefore, let's rename this variable and the constant to be more meaningful.